flake.nix

```diff
@@ -11,15 +9,20 @@
       # provide a dev shell for all systems that the holonix flake supports
       systems = builtins.attrNames holonix.devShells;
 
-      perSystem = { config, system, pkgs, ... }:
+      perSystem = { inputs', config, system, pkgs, ... }:
         {
           devShells.default = pkgs.mkShell {
-            inputsFrom = [ holonix.devShells.${system}.holochainBinaries ];
-            packages = with pkgs; [
-              # macos requires go as of late
-              (pkgs.lib.optional pkgs.stdenv.isDarwin go)
-            ];
+            packages = [
+              inputs'.holonix.packages.holochain
+              inputs'.holonix.packages.lair-keystore
+              inputs'.holonix.packages.rust
+              pkgs.go
+              pkgs.openssl
```

Comment:
Are these required on both platforms?

Reply:
Yes, Go is required to build tx5 and openssl isn't vendored at 0.3, I only changed that for 0.4 onwards